src/ln/functional_tests.rs

```diff
@@ -83,12 +83,12 @@ impl Drop for Node {
 	}
 }
 
-fn create_chan_between_nodes(node_a: &Node, node_b: &Node) -> (msgs::ChannelAnnouncement, msgs::ChannelUpdate, msgs::ChannelUpdate, [u8; 32], Transaction) {
-	create_chan_between_nodes_with_value(node_a, node_b, 100000, 10001)
+fn create_chan_between_nodes(node_a: &Node, node_b: &Node, local_features: &Option<msgs::LocalFeatures>) -> (msgs::ChannelAnnouncement, msgs::ChannelUpdate, msgs::ChannelUpdate, [u8; 32], Transaction) {
```

Comment:
It may be easier diff/rebase-wise if you just create a new intermediary wrapper function and leave the existing functions as-is. Just create a create_chan_between_nodes_with_features and make the other functions call it.

Reply:
You mean like rename the original, and let the original calls go towards towards a wrapper that puts in None?